Sources/TokamakDemo/CanvasDemo.swift

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019-2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 9/19/21.
+//
+
+import Foundation
+import TokamakShim
+
+@available(macOS 12.0, iOS 15.0, *)
+public struct CanvasDemo: View {
+  public var body: some View {
+    Confetti()
+  }
+}
+
+@available(macOS 12.0, iOS 15.0, *)
+struct Confetti: View {
+  static let colors: [Color] = [
+    Color.red,
+    Color.green,
+    Color.blue,
+    Color.pink,
+    Color.purple,
+    Color.orange,
+    Color.yellow,
+  ]
+  @State private var startDate = Date()
+  private let pieces: [Piece] = (0..<50).map { _ in
+    Piece(
+      point: CGPoint(x: CGFloat.random(in: 0...1), y: CGFloat.random(in: 0...1)),
+      size: CGSize(width: CGFloat.random(in: 0...1), height: CGFloat.random(in: 0...1)),
+      initialVelocity: CGSize(
+        width: -CGFloat.random(in: -1...1),
+        height: CGFloat.random(in: 0...2)
+      ),
+      angularVelocity: Double.random(in: -1...1),
+      color: Self.colors.randomElement()!
+    )
+  }
+
+  struct Piece {
+    let point: CGPoint
+    let size: CGSize
+    let initialVelocity: CGSize
+    let angularVelocity: Double
+    let color: Color
+  }
+
+  static let shape = Rectangle()
+
+  var body: some View {
+    TimelineView(AnimationTimelineSchedule.animation) { timeline in
+      Canvas { context, size in
+        let elapsed = CGFloat(
+          timeline.date.timeIntervalSince1970 - startDate
+            .timeIntervalSince1970
+        ) * 10
```

Comment:
It would be better if the demo looped rather than going to an empty screen after a few seconds.

```suggestion
        ).truncatingRemainder(dividingBy: 4.0) * 10
```